src/lib/components/sidebar/Sidebar.component.js

```diff
@@ -16,10 +16,11 @@ type Items = Array<Item>;
 export type Props = {
   expanded?: boolean,
   actions: Items,
+  hoverable?: boolean,
   onToggleClick?: () => void,
 };
 
-const SidebarContainer = styled.div`
+const Wrapper = styled.div`
```

Comment:
Looks like the Wrapper here doesn't include the entire height, only the items. 
I would suggest using zIndex here to make sure the sidebar always hover on the content of the page.